config/banners.yml

```diff
@@ -8,13 +8,13 @@ donate_2023:
     - [banners/donate_2023_1@2x.jpg, 2x]
   dark: true
   enddate: 2023-dec-31
-sotmeu_2023:
-  id: sotmeu_2023
-  alt: State of the Map Europe 2023
-  link: https://stateofthemap.eu/
-  img: banners/StateoftheMapEurope_2023.png
-  startdate: 2023-oct-05
-  enddate: 2023-nov-05
+sotmeu_2024:
+  id: sotmeu_2024
+  alt: State of the Map Europe 2024
+  link: https://tobilet.pl/state-of-the-map-europe-2024.html
```

Comment:
The links have generally gone to the conference page, not the individual ticket page. That would be https://stateofthemap.eu/ in this case.

Reply:
Yes, that was intentional request from organizers. @wzbartczak

Reply:
Yes, we agreed that the banner should be a direct link to purchase tickets.

Reply:
And while making it I assume that such divergence from tradition would be fine, but I have no great experience with this banners.

Though looking at https://github.com/openstreetmap/openstreetmap-website/blob/master/FAQ.md#how-do-i-create-a-banner-to-promote-my-openstreetmap-event again

> link - URL for your event page (e.g. https://supporting.openstreetmap.org/)

I guess it could be interpreted narrowly and as requesting to link the general event page only

Reply:
I think making it a link to the ticket page is a mistake, especially given the language barrier of that page being mostly in Polish.

Reply:
Thanks for pointing this out @pnorman, definitely worth double-checking. 

Since it's been confirmed by the conference organisers, and I think it's really their decision to make, I'm therefore happy to merge this as-is.